Zend/Optimizer/zend_func_infos.h

```diff
@@ -0,0 +1,49 @@
+/* This is a generated file, edit the .stub.php files instead. */
+
+static const func_info_t func_infos[] = {
+    F1("zend_version", MAY_BE_STRING),
+    FN("func_get_args", MAY_BE_ARRAY|MAY_BE_ARRAY_KEY_LONG|MAY_BE_ARRAY_OF_ANY),
+    F1("get_class_methods", MAY_BE_ARRAY|MAY_BE_ARRAY_KEY_LONG|MAY_BE_ARRAY_OF_STRING),
+    F1("get_included_files", MAY_BE_ARRAY|MAY_BE_ARRAY_KEY_LONG|MAY_BE_ARRAY_OF_STRING),
+    FN("set_error_handler", MAY_BE_STRING|MAY_BE_ARRAY|MAY_BE_ARRAY_KEY_LONG|MAY_BE_ARRAY_OF_STRING|MAY_BE_ARRAY_OF_OBJECT|MAY_BE_OBJECT|MAY_BE_NULL),
+    F0("restore_error_handler", MAY_BE_TRUE),
+    FN("set_exception_handler", MAY_BE_STRING|MAY_BE_ARRAY|MAY_BE_ARRAY_KEY_LONG|MAY_BE_ARRAY_OF_STRING|MAY_BE_ARRAY_OF_OBJECT|MAY_BE_OBJECT|MAY_BE_NULL),
+    F0("restore_exception_handler", MAY_BE_TRUE),
+    F1("get_resource_type", MAY_BE_STRING),
+    F1("get_defined_constants", MAY_BE_ARRAY|MAY_BE_ARRAY_KEY_STRING|MAY_BE_ARRAY_OF_ANY),
+    FN("socket_export_stream", MAY_BE_RESOURCE|MAY_BE_FALSE),
+    F1("gzopen", MAY_BE_RESOURCE|MAY_BE_FALSE),
+#if HAVE_NANOSLEEP
+    F1("time_nanosleep", MAY_BE_ARRAY|MAY_BE_ARRAY_KEY_STRING|MAY_BE_ARRAY_OF_LONG|MAY_BE_BOOL),
+#endif
+#if defined(PHP_WIN32) || HAVE_DNS_SEARCH_FUNC
+    F1("dns_get_record", MAY_BE_ARRAY|MAY_BE_ARRAY_KEY_LONG|MAY_BE_ARRAY_OF_ARRAY|MAY_BE_FALSE),
+#endif
+    FN("opendir", MAY_BE_RESOURCE|MAY_BE_FALSE),
+    F1("popen", MAY_BE_RESOURCE|MAY_BE_FALSE),
+    F1("fopen", MAY_BE_RESOURCE|MAY_BE_FALSE),
+    F1("tmpfile", MAY_BE_RESOURCE|MAY_BE_FALSE),
+    F1("fsockopen", MAY_BE_RESOURCE|MAY_BE_FALSE),
+    FN("pfsockopen", MAY_BE_RESOURCE|MAY_BE_FALSE),
+#if defined(PHP_CAN_SUPPORT_PROC_OPEN)
+    F1("proc_open", MAY_BE_RESOURCE|MAY_BE_FALSE),
+#endif
+    F1("stream_context_create", MAY_BE_RESOURCE),
+    FN("stream_context_get_default", MAY_BE_RESOURCE),
+    FN("stream_context_set_default", MAY_BE_RESOURCE),
+    FN("stream_filter_prepend", MAY_BE_RESOURCE|MAY_BE_FALSE),
+    FN("stream_filter_append", MAY_BE_RESOURCE|MAY_BE_FALSE),
+    F1("stream_socket_client", MAY_BE_RESOURCE|MAY_BE_FALSE),
+    F1("stream_socket_server", MAY_BE_RESOURCE|MAY_BE_FALSE),
+    F1("stream_socket_accept", MAY_BE_RESOURCE|MAY_BE_FALSE),
+    FN("zip_open", MAY_BE_RESOURCE|MAY_BE_LONG|MAY_BE_FALSE),
```

Comment:
Unfortunately, I'm not yet exactly sure when to use `@refcount 1`/`@refcount N`, so I only went with the latter for now :(